docs/index.md

```diff
@@ -0,0 +1,5 @@
+# Welcome to the G2P documentation!
+
+G2P is a tool for doing rule-based conversions for text.
+
+This website has the technical documentation for G2P, but we've also written a [7-part blog series](https://blog.mothertongues.org/g2p-background/) if you want a more thorough introduction to why G2P exists and what you can use it for.
```

Comment:
It feels very naked when you land on the home page and this is all you see, you have to scan to find that the navigation menu is on the left. I think I would add a TOC here under this paragraph, or at least put a note saying the navigation is on the left.